shared/naturalcrit/nav/nav.less

```diff
@@ -87,10 +87,15 @@ nav{
 				position         : relative;
 				display          : block;
 				width            : 100%;
-				vertical-align   : middle;
-				padding          : 8px 5px;
+				padding			 : 5px 20px 5px 12px;

```

Comment:
While you are unifying the alignments, let's also keep the top and bottom margins the same (change 5px to 8px to match the top-level button)

Reply:
Also this large 20px right padding makes any menu items without icons squeezed over in an odd way.

![image](https://github.com/naturalcrit/homebrewery/assets/5878534/6a8e2c0a-0984-42dc-a37c-e0732c0ae160)

Reply:
will add icons